client/src/pages/projects/ProjectCard.jsx

```diff
@@ -0,0 +1,61 @@
+import React from "react";
+import {
+	DiscordIcon,
+	GithubIcon,
+	ProjectCardImg,
+	ProjectIcon,
+} from "../../asset/icons/icons";
+
+function ProjectCard({
```

Comment:
Move this file to `components/projects/project-card/project-card.jsx`. It's easier to maintain and scale up if we have more subpages inside projects.